### PR TITLE
Adds renovate configuration to repo - hacktoberfest

### PR DESCRIPTION
**Pull request by joshharrison626**
# Hacktoberfest Title
Adds renovate configuration to repo.

## Description of what's changing
Uses much of the methods described in this document (https://docs.google.com/document/d/1aJ3eIpX65tgyIwPL1Pympm0Qsz-phhXVfT5RhAmXM_M/edit#) and sets up the repo to use renovate to manage dependencies.
This suggested configuration will
1. Group PR's into two groups, Major and Minor/Patch
1. It will NOT auto-merge (but you can change this at anytime by setting `automerge` to `true`
1. Allows Renovate to manage ALL dependencies
1. Uses the `chore` commit type

## What else might be impacted?
No other impact; however, this PR will not actually take affect until you merge this PR and then add the repo to the Renovate service (https://github.intuit.com/github-apps/renovate-pro/). Once both are done, you will start seeing PR's being generated by Renovate. They will not be merged automatically, and they will be in two groups to help make it easier to manage the dependency PRs: Major, and Minor/Patch.

## Which issue does this PR relate to?
#26 

## Checklist

[x] Tests are written and maintain or improve code coverage - N/A
[x] I've tested this in my application using `yarn link` (if applicable) - N/A
[x] You consent to and are confident this change can be released with no regression - N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,9 +46,10 @@ We do not allow contributors to claim issues. If you find something interesting
 1. Install dependencies locally by running `yarn` or `npm install`
 1. Make your changes
 1. Make sure it builds using `yarn build` or `npm run build`
-1. Run the tests (you added tests, right?) with `npm test` or `yarn tests`
+1. Run the tests (you added tests, right?) with `npm test` or `yarn test`
 1. Test your changes in your consuming code or using our demo project: Run [`yarn link`](https://classic.yarnpkg.com/en/docs/cli/link) or [`npm link`](https://docs.npmjs.com/cli/link)
 1. Ensure the code coverage is the same or higher than before your changes
+1. Ensure commit messages is properly formatted: `type(subject): input`. Ex: `chore(prettier): update prettier to 2.x`
```
Comment (skodamarthi) [inline README.md:52]:
```suggestion
1. Ensure commit message is properly formatted: `type(subject): input`. Ex: `chore(prettier): update prettier to 2.x`
```

Comment (skodamarthi) [reply README.md:52]: Thank you for adding this!

**Comment by skodamarthi** [comment]
@joshharrison626 Not sure why the PR build is failing. We saw this issue with another PR as well but do not exactly know the root cause yet. https://github.com/intuit/user-data-for-fraud-prevention/pull/74. Do you have any idea what could be the issue?

**Comment by reubenae** [comment]
I don't see why the CI job is failing on this PR. @skodamarthi any ideas? Can we just go ahead and merge this anyway?

**Comment by joshharrison626** [comment]
Sorry I don't have access to circle-ci to help troubleshoot the test failures.

**Comment by reubenae** [comment]
No worries @joshharrison626 - this definitely isn't an issue with your PR. We're working to fix the CI issue and I am confident merge this PR before hacktoberfest ends

**Comment by joshharrison626** [comment]
Just wanted to check in and see if there was anything you need from me for this PR. Thanks!

**Comment by skodamarthi** [comment]
Thanks @joshharrison626.. merging this PR now. I and @reubenae will do the configuration changes on Renovate to add our repo sometime next week. We will get back to you if we have any questions.

**Comment by joshharrison626** [comment]
Very cool, and yes, I'm happy to help finish the renovate setup if you need help.

**Comment by joshharrison626** [comment]
Could you please add the `hacktoberfest-accepted` label to this PR?

**Comment by reubenae** [comment]
@joshharrison626 done, though we've registered this repo for hacktoberfest so it shouldn't be required for it to show up in your stats. Thanks for the contribution!

**Pull request merged**